src/app/shared/settings/abstract-setting.component.ts

```diff
@@ -5,15 +5,17 @@ abstract class AbstractSetting extends HTMLElement {
 	canEdit = false;
 	activesValues: any;
 	separator = ',';
+	name = '';
 
 	private callback: (value: string) => void;
 
 	constructor() {
 		super();
+		this.name = this.dataset?.name || this.name;
 		this.canEdit = (this.dataset?.canEdit === 'true') || this.canEdit;
```

Comment:
Il me semblait que l’accès à `.dataset` ne fonctionnait pas côté extension ?

Reply:
Et bien j'arrive à récupérer la valeur mais pas à l'éditer. `this.dataset.name` me retourne bien ce que j'attend. Par contre `this.dataset.name = 'New name'` ne fonctionne pas.

Reply:
Okééé trop bizarre -_-